R/community.R

```diff
@@ -2782,3 +2782,44 @@ communities <- groups.communities
 #' @export
 #' @family functions for manipulating graph structure
 contract <- contract_vertices_impl
+
+
+#' Voronoi partitioning of a graph
+#'
+#' @description
+#' `r lifecycle::badge("experimental")`
+#'
+#' This function partitions the vertices of a graph based on a set of generator
+#' vertices. Each vertex is assigned to the generator vertex from (or to) which
+#' it is closest.
+#'
+#' [groups()] may be used on the output of this function.
+#'
+#' @param graph The graph to partition into Voronoi cells.
+#' @param generators The generator vertices of Voronoi cells.
+#' @inheritParams distances
+#' @param mode Character string. In directed graphs, whether to compute
+#'   distances from generator vertices to other vertices (\dQuote{out}), to
+#'   generatorvertices from other vertices (\dQuote{in}), or ignore edge
+#'   directions entirely (\dQuote{all}). Ignored in undirected graphs.
+#' @param tiebreaker Character string that specifies what to do when a vertex
+#'   is at the same distance from multiple generators. \dQuote{random} assigns
+#'   a minimal-distance generator randomly, \dQuote{first} takes the first one,
+#'   and \dQuote{last} takes the last one.
+#' @return A named list with two components:
+#'   \item{membership}{numeric vector giving the cluster id to which each vertex
```

Comment:
https://roxygen2.r-lib.org/articles/rd-formatting.html#lists

Use "*" instead of item

Reply:
So like this?

```
 * `membership` numeric vector giving ...
```

Reply:
We have this pattern everywhere else in the codebase: `@return ... \item{} ...` . I checked the rendered output, it looks good. I propose to keep it as is.